drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -975,25 +975,65 @@ def _get_user_state(self):
     def _get_correct_state(self):
         """
         Returns one of the possible correct states for the configured data.
+
+        Items dropped by user in the correct zones are retained in those zones.
+        If there are more than one correct zone of a group of items which are dropped in
+        wrong zones, they are equally distributed among the correct zones.
         """
         state = {}
         items = copy.deepcopy(self.data.get('items', []))
+
+        zone_count = {}
+        correct_items = []
+
+        def _get_display_zone(zone_count, zones):
```

Comment:
A "preferred" zone could be more accurate.

Reply:
Changed